docs/manual-indexes.adoc

```diff
@@ -0,0 +1,166 @@
+= Manual Indexes
+
+== Data Used
+
+In the below examples I used the flight data setup from the following link:
+
+https://github.com/nicolewhite/neo4j-flights
```

Comment:
perhaps we can include a subset of the load script as we plan to turn this manual into individual browser guides, then they should be self-contained.